### PR TITLE
Fix rosdep keys and use rosdep in tutorials

### PR DESCRIPTION
**Pull request by timonegk**

### CODE REVIEW

**Review by Flova — DISMISSED** [review]

File: bitbots_docs/docs/manual/tutorials/install_software_ros2.rst [hunk 1]
```diff
@@ -6,14 +6,14 @@ In this tutorial, we will learn how to install ROS2 Rolling Ridley on Ubuntu 22.
...
+- Install some more basic ROS packages: ``sudo apt install python3-rosdep python3-colcon-common-extensions python-colcon-clean``
```
Comment (Flova) [inline bitbots_docs/docs/manual/tutorials/install_software_ros2.rst:16]: I like the usage of rosdep! There are some packages like plotjuggler, where I am not sure that they are required by an subpackage.

Comment (timonegk) [reply bitbots_docs/docs/manual/tutorials/install_software_ros2.rst]: I added some packages back to be manually installed, could you check if the list looks complete to you?

File: bitbots_docs/docs/manual/tutorials/install_software_ros2.rst [hunk 2]
```diff
@@ -22,13 +22,13 @@ If you are not already using Ubuntu 22.04, consider installing it on your system
 
 **2. Download our software**
```
Comment (jaagut) [inline bitbots_docs/docs/manual/tutorials/install_software_ros2.rst:23]: Please fix the enumeration

File: bitbots_docs/docs/manual/tutorials/install_software_ros2.rst (hunk 1 above)
Comment (Flova) [reply bitbots_docs/docs/manual/tutorials/install_software_ros2.rst]: Lgtm

**Review by Flova — DISMISSED** [review]

**Review by jaagut — APPROVED** [review]

**Pull request merged**